### PR TITLE
Get individual show Trakt API

### PR DESCRIPTION
**Pull request by almazhankapan**
Get request to shows-trakt/id?type=t calls Trakt API and shows requested information for a show with trakt_id=id and type=t where type is movie or show
Output when content with trakt id=id and type=t doesn't exist: [here](https://slack-files.com/T01LN2C2L20-F01U0GUF308-523da9dec2)
Extended output when content with trakt id=id and type=t exists and poster info is available: [here](https://slack-files.com/T01LN2C2L20-F01T7LC9561-35169882ac)
If content with trakt id=id and type=t exists and poster info is unavailable, info from only Trakt API is returned
Poster info is retrieved from Tmdb API by using tmdb_id from Trakt API using chaining (second api call relies on data from the first)

### CODE REVIEW

File: back-end/app.js [hunk 1]
```diff
@@ -124,4 +135,90 @@ app.get('/shows-trakt', (req, res, next) => {
...
+  //Otherwise return extended show information 
+  }else if (req.query.type=='show'){
+    traktURL=`https://api.trakt.tv/shows/${req.params.id}`;
+    tmdbType='tv';
+  }else if (req.query.type=='movie'){
+    traktURL=`https://api.trakt.tv/movies/${req.params.id}`;
+    tmdbType='movie';
+  }else //return Bad Request error for other errors
+    res.json(badRequestError);
+  return axios
```
Comment (sam-eng) [inline back-end/app.js:164]: Would it be possible to put some sort of check before the return statement/axios call (the first of which isn't really necessary for this case, I think?). When we want to return a `badRequestError`, the program still ends up hitting the axios call and tries to send more information from there, I think.
Doing another if statement after this first one (where `traktURL` and `tmdbType` are assigned) may take care of it. So something like this maybe:
```
if (tmdbType !== undefined) {
 put the axios stuff here
}
```
image

Comment (almazhankapan) [reply back-end/app.js]: yup, just added if statement, thanks

**Review by sam-eng — APPROVED** [review]
Looks great, thanks Alma!

**Comment by almazhankapan** [comment]
> Looks great, thanks Alma!

thank you :)

**Pull request merged**